src/FluentValidation.Tests/StringEnumValidatorTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿namespace FluentValidation.Tests
+{

```

Comment:
All braces should be at the end of the previous line for consistency with the rest of the project.

Reply:
There are several places in the code where the braces are not aligned correctly. Do you mind if I format some of them too?

Reply:
Are they in files being modified by this PR? If so, then sure - that'd be great. If not then let's leave them for now, so that this PR can stay focussed on the one issue.